### ISSUE TITLE
Feature Request Flow rate

### ISSUE THREAD

**Issue by emmrichd** [jomjol/AI-on-the-edge-device#208]
Dear jomjol,

I have a hopefully "small" feature request. How about calculating the flow rate, please? Like m^3/min or l/min? (don't worry about scaling)
Of course, I could do it in my smart home software, too, but it is event driven, meaning responding only to changes after a threshold. This makes flow calculation a bit more effort.
As additional value for mqtt would probably suit most needs. 
I could the trigger event, e.g. someone taking a shower in the morning, easily.
Anyway, thanks again for this nice project!

Regards, Dieter

**Comment by jomjol** [comment]
Working on it, should be possible.

**Comment by jomjol** [comment]
Just released a new version with a rate (and a time stamp) topic. Just be aware, that currently this is still in the "rolling" branch and will move to a release with the next version.
After the update of the firmware and the html.zip you need to add the following lines in the [MQTT] section:

```
TopicRate = wasserzaehler/rate
TopicTimeStamp = wasserzaehler/timestamp
```

Just test it and let me know, if it is working!
Until then I'll close the issue.

**Comment by emmrichd** [comment]
Flashing was fine, but I don't understand how to 
"After the update of the firmware and the html.zip you need to add the following lines in the [MQTT] section: ...."  ??
There are no free lines for additional topics?

**Comment by emmrichd** [comment]
Ok, I think I understood it.
Downloaded config.ini, changed it, and uploaded it again.

**Comment by emmrichd** [comment]
Hello,
I tested it this today, this morning, it looked pretty good:
[https://familie-emmrich.de/morgen.jpeg](url)
But later, I got negative flows:
[https://familie-emmrich.de/tag.jpeg](url)
Is it possible, that you use the raw readings to calculate the flow?
Regards, Dieter

**Comment by jomjol** [comment]
Hello,
thanks for the hint. I used the right values, but before the error checkings. So even in case a wrong reading was identified, it calculated a wrong FlowRate. Will be changed with the next rolling somewhere in the evening.

Best regards,
  jomjol

**Comment by emmrichd** [comment]
Should I try again, now?

**Comment by jomjol** [comment]
It is in the rolling and I'm just about to update the released version within the next hour to v7.0.0

**Comment by AlexanderSch90** [comment]
Hey, ein Hinweis für die die updaten wäre cool im Changelog (danke für deine Arbeit!):

After the update of the firmware and the html.zip you need to add the following lines in the [MQTT] section:

TopicRate = wasserzaehler/rate
TopicTimeStamp = wasserzaehler/timestamp

Download the config.ini, changed it, and uploaded it again.
